.circleci/config.yml

```diff
@@ -51,7 +46,22 @@ jobs:
     environment:
       GO111MODULE: "on"
 
-# packet
+  # code analysis
+  static-code-analysis:
+    resource_class: medium+
+    docker:
+      - image: circleci/golang
+    steps:
+      - checkout
+      - run:
+          name: lint
+          command: |
+            go get github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
Please use a released binary instead:
https://github.com/golangci/golangci-lint/releases/download/v1.16.0/golangci-lint-1.16.0-linux-amd64.tar.gz

We will avoid ambiguous results and have reproducible error failures.

Reply:
thanks, fixed